.travis.yml

```diff
@@ -1,13 +1,35 @@
 sudo: false
 language: node_js
 script:
-  - npm run build
   - npm run test:ci
 node_js:
+  - 0.10
```

Comment:
I think you need to quote this or risk it being interpreted as "0.1" because... yaml

Reply:
Seems to have been interpreted ok by Travis as is.

Reply:
I can of course quote it though if you prefer.

Reply:
As long as it works. I recall something like this not working in the past.